src/main/java/cceclipseplugin/core/PluginManager.java

```diff
@@ -405,32 +403,33 @@ private void registerNotificationHooks() {
 					return;
 				}
 				try {
-					putFileInWarnList(pathToFile, FileDeleteNotification.class);
+					putFileInWarnList(fullPath, FileDeleteNotification.class);
 					file.delete(true, new NullProgressMonitor());
 					mm.fileDeleted(resId);
 				} catch (CoreException e) {
 					e.printStackTrace();
-					removeFileFromWarnList(pathToFile, FileDeleteNotification.class);
+					removeFileFromWarnList(fullPath, FileDeleteNotification.class);
 					showErrorAndUnsubscribe(project.getProjectID());
 				}
 			} else {
-				System.out.println("Tried to delete file that does not exist: " + pathToFile);
+				System.out.println("Tried to delete file that does not exist: " + fullPath);
 			}
 		});
 		// File.Change
 		wsManager.registerNotificationHandler("File", "Change", dataManager.getPatchManager());
 	}
 		
-	private boolean renameFile(IFile file, IPath newPath, long projId) {
+	private boolean renameFile(IFile file, IPath newFullPath, long projId) {
```

Comment:
Same here; specify what type of path this is with the variable name. (Full path is ambiguous)